pykechain/models/property_reference.py

```diff
@@ -144,7 +146,9 @@ def _retrieve_objects(self, **kwargs) -> List[user.User]:
 
         users = []
         if user_ids:
-            users = list(self._client.users(id__in=",".join(user_ids)))
+            users = list()
+            for chunk in self.chunks(user_ids):
+                users = list(self._client.users(id__in=",".join(chunk)))
```

Comment:
does not concatenate the list (possible bug?)

`extend()` is faster than concatenation

```suggestion
                users.extend(list(self._client.users(id__in=",".join(chunk))))
```